src/views/Post/CreateThread.js

```diff
@@ -1,31 +1,49 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import React, { useState } from 'react';
 import './CreateThread.css'
-import { useNavigate } from "react-router-dom";
+import useAxiosFunction from '../../hooks/useAxiosFunction'
+import axios from '../../apis/reviveme'
+import { useNavigate } from 'react-router-dom';
+
 
 const CreateThread = () => {
-    const navigate = useNavigate();
-    const [threadData, setThreadData] = useState({
+    const [data, error, loading, axiosFetch] = useAxiosFunction();
+    const navigate = useNavigate()
+    const [postData, setPostData] = useState({
         title: '',
         author:'',
         content:'',
     });
 
     const handleChange  = (event) => {
         const {name, value} = event.target
-        setThreadData({ ...threadData, [name]: value });
+        setPostData({ ...postData, [name]: value });
     };
 
     const handleSubmit = (event) =>{
         event.preventDefault()
-        // TODO: Upon submition we should create a thread via our API and add to our db
-        setThreadData({
-            ...threadData,
+        axiosFetch({
+          axiosInstance: axios,
+          method: 'POST',
+          url: '/api/v1/threads',
+          requestConfig: {
+             //TODO: change hardcoded values once we get user API running
+            data: {
+              title: postData.title,
+              content: postData.content,
+              author_id: 1,
+            }
+          }
+        })
+        
+
+        setPostData({
+            ...postData,
             title:'',
             content:'',
```

Comment:
qq: what's the purpose of setting these to a blank string?

Reply:
just some clarity on what fields this object has since they aren't referenced when updating some times